connectors/class-connector-comments.php

```diff
@@ -593,8 +593,8 @@ public function callback_transition_comment_status( $new_status, $old_status, $c
 		$this->log(
 			/* translators: %1$s: a comment author, %2$s: a post title, %3$s: a comment type */
 			_x(
-				'%1$s\'s %3$s %2$s',
-				'Comment status transition. 1: Comment author, 2: Post title, 3: Comment type',
+				'%1$s\'s %3$s on "%5$s" %2$s',
+				'Comment status transition. 1: Comment author, 2: New status, 3: Comment type, 4. Old status, 5. Post title',
```

Comment:
**suggestion:** Could you please also update the `translators:` string above with the full list of replacement variables?

Reply:
Done, good catch!